py/strato/common/log/log2text.py

```diff
@@ -237,8 +248,20 @@ def printLogs(logFiles, formatter):
     parser.add_argument("--withThreads", action="store_true", help='print process and thread name')
     parser.add_argument("-f", "--follow", action="store_true", help='follow file forever', default=False)
     parser.add_argument("-l", "--localtime", action="store_true", help='print logs in localtime (default utc)', default=False)
+    parser.add_argument("--setLocaltimeOffset", type=int, help='set custom localtime offset in hours')
+    parser.add_argument("--restoreLocaltimeOffset", action="store_true", help='restore localtime offset to machine\'s offset')
     args = parser.parse_args()
 
+    actionHappened = False
+
+    if args.setLocaltimeOffset != None:
```

Comment:
if the flag is set to something, it'll update the config file. pretty straight forward.